docs/Release_notes.rst

```diff
@@ -3,12 +3,155 @@
+
+* added example easyconfig files for 35 new software packages:
+  * Albumentations (`#15302 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15302>`_), AMPtk (`#15435 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15435>`_), arosics (`#15249 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15249>`_), CellTypist (`#15530 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15530>`_), detectron2 (`#15442 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15442>`_),
+    EigenExa (`#15234 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15234>`_), Fastaq (`#15332 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15332>`_), FFTW.MPI (`#15561 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15561>`_), FreeBarcodes (`#15350 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15350>`_), gcloud (`#15443 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15443>`_),
+    GST-plugins-bad (`#15446 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15446>`_), gsutil (`#15507 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15507>`_), GTK4 (`#15447 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15447>`_), hector (`#15397 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15397>`_), i7z (`#15236 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15236>`_), libde265 (`#15395 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15395>`_),
+    libheif (`#15395 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15395>`_), ModelTest-NG (`#15448 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15448>`_), num2words (`#15473 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15473>`_), OGDF (`#15407 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15407>`_), panito (`#15314 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15314>`_),
+    parameterized (`#15481 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15481>`_), purge_dups (`#15385 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15385>`_), redis-py (`#15475 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15475>`_), ruamel.yaml (`#15531 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15531>`_), SCGid (`#15065 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15065>`_),
+    scPred (`#15575 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15575>`_), slow5tools (`#15457 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15457>`_), smooth-topk (`#15506 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15506>`_), SPOTPY (`#15326 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15326>`_), tmap (`#4601 <https://github.com/easybuilders/easybuild-easyconfigs/pull/4601>`_), UCC (`#4291 <https://github.com/easybuilders/easybuild-easyconfigs/pull/4291>`_),
```

Comment:
`tmap` link is wrong, should be `14601`, not `4601` (same for some other PRs, like `UCC`)